docs/Migrate-from-V1-to-V2.md

```diff
@@ -1,208 +1,172 @@
-
-
-In most cases, that's all you need to do to continue using the Knockout version in V2. The additional changes that relate to the advanced functionality of using custom property editors or custom adorners are addressed below.
-
-<a id="use-react"></a>
-## To start using React version
-
-Prior to V2, React developers used our Knockout version as a black box. We did not provide a native  SurveyJS Creator component for [React](https://reactjs.org/) framework.
-
-In V2, the React version of the SurveyJS Creator is implemented as a [React component](https://reactjs.org/docs/react-component.html) and should be used in a typical [component-specific](https://reactjs.org/docs/state-and-lifecycle.html) manner. It does not mean that if you are using React in your application, all code that you have created for V1 will go into trash.
-The most code is still valid, with an exception of the code that deals with rendering and some advanced functionality, such as custom property editors.
-
-If your application uses SurveyJS Creator and you migrate from V1 to V2, make sure you use correct packages.
-You need to install `survey-creator-react`. It depends on `survey-creator-core`, `survey-react-ui`, and `survey-core` that will be installed as well.
-
-As a result, your Survey Creator component code can look as follows:
-
-
-```javascript
-import React, { Component } from "react";
-import * as SurveyCreator from "survey-creator-react";
-import "survey-core/defaultV2.css";
-import "survey-creator-core/survey-creator-core.css";
-
-class SurveyComponent extends Component {
-    constructor() {
-        super();
-        const options = { showLogicTab: true };
-        this.creator = new SurveyCreator.SurveyCreator(options);
-        // Add event handlers, survey JSON and any customization code here:
-        //...
-    }
-    render() {
-        return <SurveyCreator.SurveyCreatorComponent creator={this.creator} />;
-    }
-}
-
-export default SurveyComponent;
+```js
+// Now
+const creator = new SurveyCreator.SurveyCreator(options);
+// ...
+// Add event handlers, survey JSON, and customizations here
+// ...
+creator.render("creatorElement");
+
+// In modular applications:
+import { SurveyCreator } from "survey-creator-knockout";
+const creator = new SurveyCreator(options);
+// ...
+// Add event handlers, survey JSON, and customizations here
+// ...
+creator.render("creatorElement");
 ```
 
+## Customized Property Editors
 
+In Survey Creator V2, we changed the internal design of the Property Grid. Most of your old customization code have to be rewritten. However, the customization became easier because we use our own survey in the new design. To add a new property editor, you just need to add a property to the survey. Refer to the following help topic for more information: [Add Custom Properties to the Property Grid](https://surveyjs.io/Documentation/Survey-Creator?id=property-grid#add-custom-properties-to-the-property-grid).
```

Comment:
I changed "property" to "question" in one place, because it seemed to make more sense... But I'm not quite sure :-)

```suggestion
In Survey Creator V2, we changed the internal design of the Property Grid. You need to rewrite most of your old customization code. On the other hand, we simplified customization. The Property Grid is now a survey. To add a new property editor, you simply need to add a question to the survey. Refer to the following help topic for more information: [Add Custom Properties to the Property Grid](https://surveyjs.io/Documentation/Survey-Creator?id=property-grid#add-custom-properties-to-the-property-grid).
```